registry.k8s.io/images/k8s-staging-cluster-api-helm/images.yaml

```diff
@@ -20,3 +20,4 @@
     "sha256:3d9ad91c93b9aeb71db69174c297a420586a7ddd000705ba94fc40d9b1cf46a1": ["v0.3.1"]
     "sha256:531fbf124db369557833123277b8c8db7237749eeccd34a31a9d4e41b5e2ee45": ["v0.3.2"]
     "sha256:0fa8d800cae3b8e9b630236ea52e800a59d3a494bb0ed240b639de3e73fb2990": ["v0.4.0"]
+    "sha256:fd24e7743f824e3ede5c6f7097334c5c3ba072be504ea521da22999ea2f188e7": ["v0.4.1"]
```

Comment:
SHA `fd24e7743f82` matches cluster-api-helm-controller v0.4.1 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-helm/global/cluster-api-helm-controller@sha256:fd24e7743f824e3ede5c6f7097334c5c3ba072be504ea521da22999ea2f188e7/details?project=k8s-staging-cluster-api-helm).